### PR TITLE
Network treasury & governance folder

### LINKED ISSUE #890: Add CAF guidance for governance
# Feature overview

**In order to** help community members identify treasury opportunities
**We will** guide them towards available governance options
**So that** it's easier to suggest ideas

## Specs
https://blog.vega.xyz/announcing-the-community-adoption-fund-03878c5ed550

### PR DESCRIPTION
**Pull request by candida-d**
Closes #890. Also breaks out existing governance content into multiple pages.

### CODE REVIEW

File: docs/concepts/governance/treasury.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+Include a description of what you're intending to do with the treasury funds in your proposal's rationale so that the community can understand why and engage with your proposal.
+
+See the [tutorial](../../tutorials/proposals/asset-transfer-proposal.md) for a template and description of each field.
+
+Another option is to submit a [freeform proposal](../../tutorials/proposals/freeform-proposal.md) to suggest a non-binding idea to the community and gauge their sentiment. This would be a good option for grants or other initiatives. Raise a freeform proposal with your plan, with clear details about what you think the money should be spent on, and if there's an organisation or provider you'd like to suggest working with, details about who they are and why they, and the initiative, are a good choice. Based on feedback and voting for your freeform proposal, you can then decide to raise a proposal to transfer assets accordingly.
+
+<!--
+Wondering what the experience is for someone who wants to spend Vega on-chain, what tools to have, what kinds of competitions can they make?
+-->
```
Comment (candida-d) [inline docs/concepts/governance/treasury.md:63]:
```suggestion
```

**Review by barnabee — CHANGES_REQUESTED** [review]
Made some suggestions, also ran out of time adding links 'cos I have another call so there are a few "needs a link" type notes in there, sorry!

File: docs/concepts/governance/treasury.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+---
+import NetworkParameter from '@site/src/components/NetworkParameter';
+
+The network treasury holds assets that can be used by the community to fund initiatives. You can take part in decision-making around building and growing Vega.
+
+Treasury assets can be used to fund trading rewards, team competitions, grants, Vega protocol software and/or product development, teams building on Vega, or other opportunities.
+
+Anyone with enough VEGA tokens can vote and raise proposals to use treasury assets based on discussions in the community. 
+
+* To vote a proposal, you usually need at least 1 VEGA, but the amount may vary per proposal type. Check the [block explorer](https://explorer.fairground.wtf/network-parameters) for each `minVoterBalance`.
```
Comment (edd) [inline docs/concepts/governance/treasury.md:16]:
```suggestion
* To vote on a proposal, you usually need at least 1 VEGA, but the amount may vary per proposal type. Check the [block explorer](https://explorer.fairground.wtf/network-parameters) for each `minVoterBalance`.
```

File: docs/concepts/governance/treasury.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+Read more about how these work in the [teams and games](../trading-on-vega/discounts-rewards.md#teams-and-games) section.
+
+**Grants and other initiatives**: A transfer can send to any Vega key, even if it's an individual and not a reward account. You can use a freeform proposal to gauge the community's interest and hone the plan before setting up the proposal for transferring assets.
+
+## How to allocate the treasury
+
+### Use governance to spend assets
+
+Assets can only be spend from the on-chain treasury by governance, when a **[proposal to transfer assets](../assets/transfers.md#governance-initiated-transfers)** is enacted.
```
Comment (edd) [inline docs/concepts/governance/treasury.md:57]:
```suggestion
Assets can only be spent from the on-chain treasury by governance, when a **[proposal to transfer assets](../assets/transfers.md#governance-initiated-transfers)** is enacted. 
```

File: docs/concepts/governance/treasury.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+**Grants and other initiatives**: A transfer can send to any Vega key, even if it's an individual and not a reward account. You can use a freeform proposal to gauge the community's interest and hone the plan before setting up the proposal for transferring assets.
+
+## How to allocate the treasury
+
+### Use governance to spend assets
+
+Assets can only be spend from the on-chain treasury by governance, when a **[proposal to transfer assets](../assets/transfers.md#governance-initiated-transfers)** is enacted. 
+
+The most direct way to propose spending the assets is by submitting one of these proposals. In practice, community members almost always discuss proposals on the **[Vega community forums ↗](https://community.vega.xyz/) before submitting**, in order to incorporate community feedback into the proposal and ensure there's broad support before going through the effort of putting it on-chain.
```
Comment (edd) [inline docs/concepts/governance/treasury.md:59]:
```suggestion
The most direct way to propose spending the assets is by submitting one of these proposals. In practice, community members almost always discuss proposals on the  **[Vega community forums ↗](https://community.vega.xyz/) before submitting**, in order to incorporate community feedback into the proposal and ensure there's broad support before going through the effort of putting it on-chain.
```

File: versioned_docs/version-v0.74/concepts/governance/treasury.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+**Grants and other initiatives**: A transfer can send to any Vega key, even if it's an individual and not a reward account. You can use a freeform proposal to gauge the community's interest and hone the plan before setting up the proposal for transferring assets.
+
+## How to allocate the treasury
+Allocating assets in the treasury requires using governance. There are a few options for moving those assets. 
+
+Before submitting a proposal, make sure you understand the full [governance lifecycle](./lifecycle.md).
+
+### Use governance to spend assets
+
+Assets can only be spend from the on-chain treasury by governance, when a [proposal to transfer assets](../assets/transfers.md#governance-initiated-transfers) is enacted.
```
Comment (edd) [inline versioned_docs/version-v0.74/concepts/governance/treasury.md:60]:
```suggestion
Assets can only be spent from the on-chain treasury by governance, when a [proposal to transfer assets](../assets/transfers.md#governance-initiated-transfers) is enacted. 
```

File: versioned_docs/version-v0.74/concepts/governance/treasury.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+---
+import NetworkParameter from '@site/src/components/NetworkParameter';
+
+The network treasury holds assets that can be used by the community to fund initiatives. You can take part in decision-making around building and growing Vega.
+
+Treasury assets can be used to fund trading rewards, team competitions, grants, Vega protocol software and/or product development, teams building on Vega, or other opportunities.
+
+Anyone with enough VEGA tokens can vote on and submit proposals to [allocate treasury assets](#how-to-allocate-the-treasury) based on discussions in the community.
+
+* To vote a proposal, you need at least 1 VEGA, but the minimum amount can vary per proposal type. Check the [block explorer ↗](https://explorer.fairground.wtf/network-parameters) for each `minVoterBalance`.
```
Comment (edd) [inline versioned_docs/version-v0.74/concepts/governance/treasury.md:16]:
```suggestion
* To vote on a proposal, you need at least 1 VEGA, but the minimum amount can vary per proposal type. Check the [block explorer ↗](https://explorer.fairground.wtf/network-parameters) for each `minVoterBalance`.
```

File: versioned_docs/version-v0.74/concepts/governance/treasury.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+## How to allocate the treasury
+Allocating assets in the treasury requires using governance. There are a few options for moving those assets. 
+
+Before submitting a proposal, make sure you understand the full [governance lifecycle](./lifecycle.md).
+
+### Use governance to spend assets
+
+Assets can only be spend from the on-chain treasury by governance, when a [proposal to transfer assets](../assets/transfers.md#governance-initiated-transfers) is enacted. 
+
+The most direct way to propose spending the assets is by submitting one of these proposals. In practice, community members almost always **discuss proposals on the[Vega community forums ↗](https://community.vega.xyz/) before submitting**, in order to incorporate community feedback into the proposal and ensure there's broad support before going through the effort of putting it on-chain.
```
Comment (edd) [inline versioned_docs/version-v0.74/concepts/governance/treasury.md:62]:
```suggestion
The most direct way to propose spending the assets is by submitting one of these proposals. In practice, community members almost always **discuss proposals on the [Vega community forums ↗](https://community.vega.xyz/) before submitting**, in order to incorporate community feedback into the proposal and ensure there's broad support before going through the effort of putting it on-chain.
```

**Pull request merged**